planning/behavior_path_planner/behavior_path_planner_path_generation.md

```diff
@@ -0,0 +1,97 @@
+## Path Generation
+
+This document explains how the path is ganerated for lane change and avoidance, etc. The implementation can be found in [path_shifter.hpp](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/include/behavior_path_planner/scene_module/utils/path_shifter.hpp).
```

Comment:
```suggestion
This document explains how the path is generated for lane change and avoidance, etc. The implementation can be found in [path_shifter.hpp](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/include/behavior_path_planner/scene_module/utils/path_shifter.hpp).
```

Reply:
solved in [4522729](https://github.com/autowarefoundation/autoware.universe/pull/2428/commits/4522729987b25407946e2da66d17c40e7af75ece)